wsapi.go

```diff
@@ -379,6 +382,20 @@ func (s *Session) UpdateStatus(idle int, game string) (err error) {
 	return s.UpdateStreamingStatus(idle, game, "")
 }
 
+//UpdateListeningStatus is used to set the user to "Listening to..."
+//If game!="" then set to what user is listening to
+//Else, set user to active and no game.
+func (s *Session) UpdateListeningStatus(idle int, game string) (err error) {
+	return s.UpdateStatusComplex(UpdateStatusData{
```

Comment:
Sorry, I was misunderstanding what you're trying here, this should be:

var g Game
if game != "" {
  g = &Game{
    Name: game,
    Type: GameTypeListening,
    URL:  "",
  }
}
return s.UpdateStatusComplex(UpdateStatusData{
  Status: "online",
  Game: g,
})